### PR TITLE
Feature/question statement displayer

### PR DESCRIPTION
**Pull request by KGomes27**
New component that displays the question statement and adapts to questions with or without images.
The component has an optional property called _placeholderImage_, that allows you to send in any image as the placeholder or use the default loader. It also has mixins for both the text and the image that make up the question. 

[Task Ticket](https://trello.com/c/WPjXQDre/30-3-crear-componente-question-statement-displayer).

### CODE REVIEW

File: src/components/question-statement-displayer.html [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+      * | Custom property            | Description                                              | Default |
+      * |----------------------------|----------------------------------------------------------|---------|
+      * | --question-statement-text  | Mixin applied to the text of the question                | {}      |
+      * | --question-statement-image | If the question has an image this mixin is applied to it | {}      |
+      *
+      * @customElement
+      * @polymer
+    */
+    class QuestionStatementDisplayer extends (class extends Polymer.Element {}) {
+      static get is() { return 'question-statement-displayer'; }
```
[the remark this answers is not in the document]
Comment (KGomes27) [reply src/components/question-statement-displayer.html]: I didn't get an error when running the linter mmm

File: src/components/question-statement-displayer.html [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+      static get properties() {
+        return {
+          /** Object containing the title of the question and an optional image */
+          question: Object,
+          /** Id of the question being displayed */
+          questionId: String,
+          // URL of the image shown on iron-image placeholder
+          placeholderImage: {
+            type: String,
+            value: '../images/placeholder.gif'
```
[the remark this answers is not in the document]
Comment (KGomes27) [reply src/components/question-statement-displayer.html]: But is `roothPath`available globally?

File: src/components/question-statement-displayer.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (pepocivs) [reply src/components/question-statement-displayer.html]: Maybe is a windows issue...  `.eslintsrc` should work. @alvarogonzalezDEV do u have this issue?

File: src/components/question-statement-displayer.html (hunk 2 above)
[the remark this answers is not in the document]
Comment (pepocivs) [reply src/components/question-statement-displayer.html]: Yes, should be

**Pull request merged**